src/main/java/fr/minuskube/inv/ClickableItem.java

```diff
@@ -16,29 +16,41 @@
 
 package fr.minuskube.inv;
 
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+import org.bukkit.Material;
 import org.bukkit.entity.Player;
 import org.bukkit.event.inventory.InventoryClickEvent;
 import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
-import java.util.function.Consumer;
-import java.util.function.Predicate;
-
-@SuppressWarnings({ "unchecked" })
-public class ClickableItem {
+@SuppressWarnings("unchecked")
+public final class ClickableItem {
 
     /**
      * ClickableItem constant with no item and empty consumer.
      */
-    public static final ClickableItem NONE = empty(null);
-
+    public static final ClickableItem NONE = ClickableItem.empty(new ItemStack(Material.AIR));
```

Comment:
Why make it air, don't waste processor power creating a new ItemStack with ItemMeta if null works just like fine